roma/romadom.php

```diff
@@ -2096,7 +2099,7 @@ public function setDocLanguage( $szDocLanguage )
     protected function callGarage ( &$garageResult, $target) {
 	$OXG = oxgarage_server . '/Conversions/ODD%3Atext%3Axml/ODDC%3Atext%3Axml/';
 	$this->getDocLanguage( $szDocLanguage );
-	$properties ="?properties=" . urlencode("<conversions><conversion index='0'><property id='pl.psnc.dl.ege.tei.profileNames'>tei</property></conversion><conversion index='1'><property id='pl.psnc.dl.ege.tei.profileNames'>tei</property><property id='oxgarage.textOnly'>true</property><property id='oxgarage.lang'>" . $szDocLanguage .  "</property></conversion></conversions>");
+	$properties ="?properties=<conversions><conversion%20index='0'><property%20id='pl.psnc.dl.ege.tei.profileNames'>tei</property></conversion><conversion%20index='1'><property%20id='pl.psnc.dl.ege.tei.profileNames'>tei</property><property%20id='oxgarage.textOnly'>true</property><property%20id='oxgarage.lang'>" . $szDocLanguage .  "</property></conversion></conversions>";
```

Comment:
This seems like the reimport of some old code?
I know because I changed it in 
https://github.com/TEIC/Roma/commit/a32203e47395d39c0c810e6f93851fe1f382a729 ;)